src/models/cart.dao.js

```diff
@@ -0,0 +1,65 @@
+const { appDataSource } = require("./data-source");
+
+const createCart = async (quantity, userId, productOptionId) => {
+  await appDataSource.query(
```

Comment:
```suggestion
  return await appDataSource.query(
```
해당 함수에서 sql query를 실행하고 더이상 처리할 로직이 없다면 return으로 맺음을 지어주세요.